modules/plot/test/spec.coffee

```diff
@@ -243,15 +254,24 @@ describe "plot", ->
       checkOptionAndSetterGetter(hx.BarSeries, 'group', ['group1', 'group2'])
 
     describe "render", ->
-      it 'should display when the height is positive', ->
```

Comment:
Looking at the other test, I definitely think this test should be left in.

The newly added test is checking that the fluent api works, it might be worth adding further test(s) to make sure the fluid graph api works.